src/common/client-app.ts

```diff
@@ -69,8 +74,8 @@ export class App {
 
   onInterval= () => {
     var current = this.boardVM.toPlain();
-    var myChanges = rfc6902.createPatch(this.shadow, current);
-    if (myChanges.length) {
+    var myChanges = rfc6902.createPatch(this.shadow, current);
+    if (myChanges.length) {
```

Comment:
I was talking about this transition:

![2016-05-03_121921](https://cloud.githubusercontent.com/assets/1157864/14988268/84b5ec70-1129-11e6-9dd8-8ace505b050e.png)